redis-snapshot-replicator/main.tf

```diff
@@ -167,21 +180,46 @@ EOF
 }
 
 resource "aws_iam_role_policy_attachment" "attach_redis_lambda_copy_policy_to_role" {
+  count      = "${var.enable ? 1 : 0}"
   role       = "${aws_iam_role.iam_for_lambda_redis.name}"
   policy_arn = "${aws_iam_policy.redis_lambda_copy.arn}"
 }
 
+resource "aws_iam_policy" "redis_lambda_remove" {
+  count = "${var.enable ? 1 : 0}"
+  name  = "${var.name}-lambda-remove-${var.environment}"
+
+  policy = <<EOF
```

Comment:
Not really an issue, but cleaner if you use an `aws_iam_policy_document` data provider.

Reply:
and perhaps also use the `aws_iam_role_policy` instead